scripts/testnet/include/nodes.sh

```diff
@@ -149,12 +147,10 @@ assembleCommand_startValidatorNode() {
   VALIDATOR_INDEX=$1
   let "PORT = $PORT_ORIGIN_VALIDATOR + $VALIDATOR_INDEX"
   let "RESTAPIPORT=$PORT_ORIGIN_VALIDATOR_REST + $VALIDATOR_INDEX"
-  let "KEY_INDEX=$VALIDATOR_INDEX"
   WORKING_DIR=$TESTNETDIR/node_working_dirs/validator$VALIDATOR_INDEX
 
   local nodeCommand="./node \
         -port $PORT -log-save -log-level $LOGLEVEL -rest-api-interface localhost:$RESTAPIPORT \
-        -tx-sign-sk-index $KEY_INDEX -sk-index $KEY_INDEX \
```

Comment:
Also here: Should revert the `-sk-index $KEY_INDEX \` part as this is used for the block sign key

Reply:
done.